addons/packages/ako-operator/1.4.0/README.md

```diff
@@ -42,3 +129,81 @@ The following configuration values can be set to customize the ako-operator inst
 ## Usage Example
 
 This walkthrough guides you through using ako-operator...
+
+### Verify your configurations
+
+After the package is installed and reconcile successfully, two AKODeploymentConfigs are deployed globally.  
+You can list and inspect them with `kubectl get akodeploymentconfig`. They are the configuration interfaces for the ako-operator.
+In general,
+
+* `install-ako-for-management-cluster` specifies the parameters to deploy AKO statefulset in the management cluster, which
+  then provides L4/L7 load balancing for applications.
+* `install-ako-for-all` specifies the parameters to deploy AKO statefulsets and reconcile users in each future workload clusters. The ako-operator
+  passes these parameters to each workload clusters via addon secrets.
+
+In the management cluster, ensure that there is no error level message in the ako-operator manage log.
+
+```shell
+# retrieve the pod name of the ako-operator manager
+kubectl get pods -n tkg-system-networking
+
+# check if the log is error-free.
+kubectl logs ako-operator-controller-manager-xxxxxxxx-xxxxx manager -n tkg-system-networking
+```
+
+## Usage Example
+
+### Create an L4 Service in management cluster
+
+Deploy a `LoadBalancer` type of service with the following manifest.
+
+```shell
+$ cat <<EOF >sample-svc.yaml
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: corgi-test
+  labels:
+    corgi: test
+spec:
+  selector:
+    matchLabels:
+      corgi: test
+  template:
+    metadata:
+      labels:
+        corgi: test
+    spec:
+      containers:
+        - name: nginx
+          image: harbor-repo.vmware.com/dockerhub-proxy-cache/library/nginx
+          ports:
+            - containerPort: 80
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: corgi-test
+spec:
+  type: LoadBalancer
+  selector:
+    corgi: test
+  ports:
+    - nodePort: 30008
+      port: 80
+      targetPort: 80
+EOF
+```
+
+After that, verify that the service has been assigned an external IP using `kubectl`. Visit the AVI controller
+dashboard at `http://<avi_controller_ip>` and ensure a virtual service has been created accordingly for this deployment.
```

Comment:
`avi_controller_ip` is not accurate, sometimes users may have an FQDN for AVI controller.